media-sound/spotify_qt/spotify_qt-2.4.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A Spotify client using Qt"
+DESCRIPTION="A lighter alternative to the official client, inspired by \
+spotify-tui.
+Much like spotify-tui, you need an actual Spotify client running, for example \
+spotifyd, which can be configured from within the app. Also like other \
+clients, controlling music playback requires Spotify Premium."
+HOMEPAGE="https://github.com/kraxarn/spotify-qt"
+COPYRIGHT="2020 Kraxarn"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/kraxarn/spotify-qt/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="ca9b4af238f4b7392524c93bee4093db656cf240f5ef490c6f569238dec35455"
+SOURCE_FILENAME="spotify-qt-v$portVersion.tar.gz"
+SOURCE_DIR="spotify-qt-$portVersion"
+ADDITIONAL_FILES="spotify-qt.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	spotify_qt$secondaryArchSuffix = $portVersion
+	cmd:spotify_qt
+#	app:spotify_qt
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5DBus$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5DBus$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:qdbuscpp2xml$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=/appsDir
```

Comment:
$appsDir probably?

Reply:
Actually even that would be incorrect as "make install" doesn't work, but probably should refer to $prefix/bin rather then $appsDir :) (but yes a typo) :)